packages/asset-buyer/package.json

```diff
@@ -38,8 +38,9 @@
     "homepage": "https://github.com/0xProject/0x-monorepo/packages/asset-buyer/README.md",
     "dependencies": {
         "@0x/assert": "^2.1.3",
+        "@0x/asset-swapper": "^1.0.1",
```

Comment:
How does `asset-buyer` use `asset-swapper`?

Reply:
Oops, this is leftover. Will remove